kivy/core/window/__init__.py

```diff
@@ -523,7 +523,7 @@ def _get_height(self):
     def _get_center(self):
         return self.width / 2., self.height / 2.
 
-    center = AliasProperty(_get_center, bind=('width', 'height'), cache=True)
+    center = AliasProperty(_get_center, bind=('width', 'height'))  # Do not cache center
```

Comment:
The problem seems to have been that density is changed, but the property being cached was not updated. Is that correct?

If so, rather than not caching which has performance implications, a better solution would be to make density a `NumericProperty`, and then add it to the bind list.